k8s-yaml-templates/quickstart/cwagent-fluentd-quickstart.yaml

```diff
@@ -99,7 +99,7 @@ spec:
     spec:
       containers:
         - name: cloudwatch-agent
-          image: amazon/cloudwatch-agent:1.230621.0
+          image: public.ecr.public/cloudwatch-agent/cloudwatch-agent:1.230621.0
```

Comment:
Are there any reasons why we are not having latest or any recent version? Same for other case too.